### PR TITLE
Increase input buffer size

### PR DESCRIPTION
**Pull request by kanes115**

### CODE REVIEW

File: lib/membrane_bin_rtp/receiver.ex [hunk 1]
```diff
@@ -55,8 +56,14 @@ defmodule Membrane.Bin.RTP.Receiver do
     parser_ref = {:parser, make_ref()}
 
     children = [{parser_ref, RTP.Parser}]
+    warn_buf_size = (0.5 * @bin_buffer_size) |> trunc() |> Kernel.+(1)
```
Comment (mat-hek) [inline lib/membrane_bin_rtp/receiver.ex]: e?

Comment (kanes115) [reply lib/membrane_bin_rtp/receiver.ex]: e

File: lib/membrane_bin_rtp/receiver.ex [hunk 2]
```diff
@@ -55,8 +56,14 @@ defmodule Membrane.Bin.RTP.Receiver do
     parser_ref = {:parser, make_ref()}
 
     children = [{parser_ref, RTP.Parser}]
+    warn_buf_size = div(@bin_buffer_size, 2)
 
-    links = [link_bin_input(pad) |> to(parser_ref) |> to(:ssrc_router)]
+    links = [
+      link_bin_input(pad)
+      |> via_in(:input, buffer: [warn_size: warn_buf_size, fail_size: @bin_buffer_size])
```
Comment (mat-hek) [inline lib/membrane_bin_rtp/receiver.ex]: store entire buffer params list in the moduledoc

File: lib/membrane_bin_rtp/receiver.ex [hunk 3]
```diff
@@ -16,6 +16,7 @@ defmodule Membrane.Bin.RTP.Receiver do
 
   @static_fmt_file "rtp-parameters-1.csv" |> Path.expand(__DIR__)
   @bin_buffer_size 500
+  @bin_buffer_size_warn 250
```
Comment (mat-hek) [inline lib/membrane_bin_rtp/receiver.ex]: I meant `@bin_input_buffer_params [warn_size: 250, fail_size: 500]`

**Pull request merged**